packages/web-config-server/src/apiV1/DataAggregatingRouteHandler.js

```diff
@@ -19,13 +18,25 @@ export class DataAggregatingRouteHandler extends RouteHandler {
     this.aggregator = createAggregator(Aggregator, this);
   }
 
-  // Builds the list of entities data should be fetched from, using org unit descendents of the
+  // Builds the list of entities data should be fetched from, using org unit descendants of the
```

Comment:
@edmofro, this is also a potential merge conflict, just fyi

Reply:
Yes it is. Once merged, I'll get you to update the entity hierarchy performance branch to match your logic changes

Reply:
Resolved - you might like to check it out at https://github.com/beyondessential/tupaia/pull/1484/files#diff-73816ca80b10483285bacd692bd6ad334a606a0a2f287201d4a864de4774cbc8R15

Reply:
Thanks for doing that @edmofro, and sorry that I let that task slip by 😬.

Reply:
No worries